src/applications/simple-forms/20-10207/tests/e2e/veteran-story.cypress.spec.js

```diff
@@ -409,10 +413,231 @@ testSuite('PP 10207 - Veteran', () => {
         pagePathIsCorrect('mailing-address');
       });
 
-      it('advances to next-page when NO is selected', () => {
+      it('advances to Your-phone-and-email-address page when NO is selected', () => {
         cy.contains('No').click();
         continueToNextPage();
-        // TODO: Change path to next-page once built
+        pagePathIsCorrect('phone-and-email');
+      });
+    });
+
+    describe('Mailing-address page', () => {
```

Comment:
it's awesome your being so thorough with testing 🙌 
I think in the future we should still consider what should go in a unit test and what should go in e2e. since e2e is much more expensive

Reply:
Yeah, it's still early days for the e2e/qa process &mdash; right now my add'l spec's CI-skipped.  I'll need to sync-up with Tricia to try pruning her TestRail test-cases down.